hello/lib/hello_web/templates/layout/admin.html.heex

```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8"/>
+    <meta http-equiv="X-UA-Compatible" content="IE=edge"/>
+    <meta name="viewport" content="width=device-width, initial-scale=1.0"/>
+    <%= csrf_meta_tag() %>
+    <%= live_title_tag assigns[:page_title] || "Hello", suffix: " · Phoenix Framework" %>
+    <link phx-track-static rel="stylesheet" href={Routes.static_path(@conn, "/assets/app.css")}/>
+    <script defer phx-track-static type="text/javascript" src={Routes.static_path(@conn, "/assets/app.js")}></script>
+  </head>
+  <body>
+    <header>
+      <section class="container">
+        <nav>
+          <ul>
+            <li><a href="https://hexdocs.pm/phoenix/overview.html">Get Started</a></li>
+            <%= if function_exported?(Routes, :live_dashboard_path, 2) do %>
```

Comment:
@Vretamoza could you please explain in deep `function_exported?/3`?

Reply:
It receives three parameters, a module, a function and its arity. The function returns true if the given module is loaded and has a function with the given arity. If that's not the case, it reurns false.

Reply:
And why use it? what is the reason behind it?

Reply:
It is used to check if the module and the function I'm going to use is loaded, but if it's not, it doesn't load it.